### PR TITLE
Fix some small issue in cdn swagger

### PR DESCRIPTION
**Pull request by csmengwan**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@csmengwan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (qinwa). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: arm-compute/2015-06-15/swagger/ComputeCompositeClient.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (amarzavery) [inline arm-compute/2015-06-15/swagger/ComputeCompositeClient.json]: @csmengwan - Why do you have this file for compute?

File: arm-cdn/2015-06-01/swagger/cdn.json [hunk 2]
```diff
@@ -250,6 +250,12 @@
           }
         ],
         "responses": {
+		  "200": {
```
Comment (amarzavery) [inline arm-cdn/2015-06-01/swagger/cdn.json]: tabbing

File: arm-cdn/2015-06-01/swagger/cdn.json [hunk 3]
```diff
@@ -726,7 +735,10 @@
         ],
         "responses": {
           "202": {
-            "description": "Accepted and the operation will complete asynchronously."
+            "description": "Accepted and the operation will complete asynchronously.",
+			"schema": {
```
Comment (amarzavery) [inline arm-cdn/2015-06-01/swagger/cdn.json]: indent 2 spaces

File: arm-cdn/2015-06-01/swagger/cdn.json [hunk 4]
```diff
@@ -676,7 +682,10 @@
...
```
Comment (amarzavery) [inline arm-cdn/2015-06-01/swagger/cdn.json]: indent 2 spaces every where

**Pull request merged**